build-windows/libetpan_version.h

```diff
@@ -38,26 +38,13 @@
 #endif
 
 #ifndef LIBETPAN_VERSION_MINOR
-#define LIBETPAN_VERSION_MINOR 8
+#define LIBETPAN_VERSION_MINOR 6
```

Comment:
Curious why we had to decrease this and remove the other defs below?

Reply:
The configure script makes this change for some reason. I would expect that building (and hence running configure) should only modify files in the build folder or files that are not tracked by Git, but that doesn't seem to hold true here.

I can't remember, but I think it only happened when built from the embedded version in the apps repo. So if I didn't put the change here, every time we updated libetpan via carthage the next build would require committing another change as well in apps, so I changed it here.

I must admit I am a bit sketched out by the need for this (particularly since it appears to be a downgrade even though I grabbed the latest libetpan version), and the main thing that made we shrug my shoulders and move on is that this is a change in the windows version and not the mac (which makes it even more strange why a mac configure would modify it).

Reply:
That sounds strange, but yeah, doesn't affect us so no big deal. We can always clean it up before upstreaming if we need to.